### ISSUE TITLE
Backport request: 1.19.4

### ISSUE THREAD

**Issue by Mr-Jokes-3** [Superkat32/YouWereSlain#8]
Are there ideas to backport to 1.19.4?

**Comment by Superkat32** [comment]
Yes! I do plan on eventually porting this mod all the way to 1.19.2, from the latest available version. My current plan is to make a bunch of improvements to this mod, then backport it.

**Comment by normalUserKarn** [comment]
I am looking for this too, the mod looks amazing!
